### PR TITLE
remove dark mode

### LINKED ISSUE #632: Remove dark mode
Removing dark mode in the sketch page
image
No use of using dark mode in sketch so we will remove it.

### PR DESCRIPTION
**Pull request by ashuvssut**
closes #632 

------------------------
_- I have participated in GSSOC_

### CODE REVIEW

File: src/pages/Sketch/components/Canvas.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React, {useState, useEffect, useRef, useCallback} from 'react';
 import styles from './Canvas.module.css';
 import Toolbox from './Toolbox/Toolbox';
-import {FaPencilAlt, FaRegSquare, FaDownload, FaRegCircle, FaSlash, FaRegMoon, FaSun, FaFont} from 'react-icons/fa';
+import {FaPencilAlt, FaRegSquare, FaDownload, FaRegCircle, FaSlash, /*FaRegMoon, FaSun,*/ FaFont} from 'react-icons/fa';
```
Comment (anushbhatia) [inline src/pages/Sketch/components/Canvas.js:4]: remove the comments too. we don't want that

File: src/pages/Sketch/components/Canvas.js [hunk 2]
```diff
@@ -42,9 +42,10 @@ function Canvas() {
     const [typeState, setTypeState] = useState(null);
     const [downPoint, setDownPoint] = useState({x: "", y: ""});
     const [mousePosition, setMousePosition] = useState({x: "0", y: "0"});
-    const [isDarkModeOn, setIsDarkMode] = useState(
-        localStorage.getItem("theme") === "true" ? true : false
-    );
+    // const [isDarkModeOn, setIsDarkMode] = useState(
```
Comment (anushbhatia) [inline src/pages/Sketch/components/Canvas.js:45]: here too

**Comment by ashuvssut** [comment]
@anushbhatia I have given comments so that in future if you decide to restore the darktheme again then you have to just remove 3 comments(as given in my PR)

File: src/pages/Sketch/components/Canvas.js (hunk 1 above)
Comment (ashuvssut) [inline src/pages/Sketch/components/Canvas.js:4]: ok I will remove these two icons. Bcz adding these again (when restoring darktheme) is not such a big problem

**Comment by anushbhatia** [comment]
no use of dark mode in the sketch page who wants their drawing page to be black🤣🤣

**Comment by ashuvssut** [comment]
> no use of dark mode in the sketch page who wants their drawing page to be black🤣🤣

ok got it

**Comment by ashuvssut** [comment]
thanks for clarifying

**Comment by ashuvssut** [comment]
removed dark theme from all js and css

**Comment by ashuvssut** [comment]
Sir I have not participated in crossWoC😅

so no need to put crosswoc label

**Pull request merged**